compiler/MetadataUtils.ts

```diff
@@ -363,7 +363,10 @@ export default class MetadataUtils {
         '*://*.clashofclans.com/*',
         // https://github.com/AdguardTeam/PopupBlocker/issues/238
         '*://online.bfgruppe.de/*',
-        '*://*.online.bfgruppe.de/*'
+        '*://*.online.bfgruppe.de/*',
+        // https://github.com/AdguardTeam/PopupBlocker/issues/238
```

Comment:
I didn't notice it before, but it seems that there is a wrong link to issue.
It should be - `https://github.com/AdguardTeam/PopupBlocker/issues/241`
The same here - https://github.com/AdguardTeam/PopupBlocker/pull/242/files#diff-2ab9c222fff241d52e1a592d1afbe9e017e794544870782a67933fe1e228434dR726

Reply:
fixed both links